### PR TITLE
VALIDATION image-alt-text-duplicated

### PR DESCRIPTION
**Pull request by mSarts**
No issue number.

### CODE REVIEW

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ The ASP.NET MVC framework with any modern database including Microsoft SQL Serve
 
 Create a new database by right-clicking the App\_Data folder in the Solution Explorer window and selecting the menu option **Add, New Item**. In the **Add New Item** dialog, select the **Data** category and the **SQL Server Database** template (see Figure 4). Name the new database ContactManagerDB.mdf and click the OK button.
 
-[![The New Project dialog box](iteration-1-create-the-application-cs/_static/image4.jpg)](iteration-1-create-the-application-cs/_static/image7.png)
+[![Screenshot shows creating a new Microsoft SQL Server Express database.](iteration-1-create-the-application-cs/_static/image4.jpg)](iteration-1-create-the-application-cs/_static/image7.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md:128]: Let's spell out "SQL" as "Sequel"

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-vb.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ The ASP.NET MVC framework with any modern database including Microsoft SQL Serve
 
 Create a new database by right-clicking the App\_Data folder in the Solution Explorer window and selecting the menu option **Add, New Item**. In the **Add New Item** dialog, select the **Data** category and the **SQL Server Database** template (see Figure 4). Name the new database ContactManagerDB.mdf and click the OK button.
 
-[![The New Project dialog box](iteration-1-create-the-application-vb/_static/image4.jpg)](iteration-1-create-the-application-vb/_static/image7.png)
+[![Screenshot shows creating a new Microsoft SQL Server Express database.](iteration-1-create-the-application-vb/_static/image4.jpg)](iteration-1-create-the-application-vb/_static/image7.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-vb.md:128]: Let's spell out "Sequel"

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md [hunk 3]
```diff
@@ -41,15 +41,15 @@ We build the application over multiple iterations. With each iteration, we gradu
 
 The goal of this iteration is to improve the appearance of the Contact Manager application. Currently, the Contact Manager uses the default ASP.NET MVC view master page and cascading style sheet (see Figure 1). These don t look bad, but I don t want the Contact Manager to look just like every other ASP.NET MVC website. I want to replace these files with custom files.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-cs/_static/image1.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image1.png)
+[![Screenshot shows the default appearance of an ASP.NET MVC Application.](iteration-2-make-the-application-look-nice-cs/_static/image1.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image1.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:44]: Let's do "ASP dot net"

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md [hunk 4]
```diff
@@ -41,15 +41,15 @@ We build the application over multiple iterations. With each iteration, we gradu
...
 
 **Figure 01**: The default appearance of an ASP.NET MVC Application ([Click to view full-size image](iteration-2-make-the-application-look-nice-cs/_static/image2.png))
 
 In this iteration, I discuss two approaches to improving the visual design of our application. First, I show you how to take advantage of the ASP.NET MVC Design gallery to download a free ASP.NET MVC design template. The ASP.NET MVC Design gallery enables you to create a professional web application without doing any work.
 
 I decided to not use a template from the ASP.NET MVC Design gallery for the Contact Manager application. Instead, I had a custom design created by a professional design firm. In the second part of this tutorial, I explain how I worked with a professional design company to create the final ASP.NET MVC design.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-cs/_static/image2.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image3.png)
+[![Screenshot shows the ASP.NET MVC Design Gallery.](iteration-2-make-the-application-look-nice-cs/_static/image2.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image3.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:52]: Let's do "ASP dot net"

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md [hunk 5]
```diff
@@ -62,21 +62,21 @@ As I write this tutorial, the most popular design in the gallery is a design nam
...
 
 **Figure 03**: Unblocking a file downloaded from the web ([Click to view full-size image](iteration-2-make-the-application-look-nice-cs/_static/image6.png))
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-cs/_static/image4.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image7.png)
+[![Screenshot shows overwriting files in the Solution Explorer.](iteration-2-make-the-application-look-nice-cs/_static/image4.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image7.png)
 
 **Figure 04**: Overwriting files in the Solution Explorer ([Click to view full-size image](iteration-2-make-the-application-look-nice-cs/_static/image8.png))
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-cs/_static/image5.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image9.png)
+[![Screenshot shows replacing ProjectName with ContactManager.](iteration-2-make-the-application-look-nice-cs/_static/image5.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image9.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:73]: Let's add a space between Project and Name and also Contact and Manager to make sure it's read correctly by the screen reader.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md [hunk 6]
```diff
@@ -62,21 +62,21 @@ As I write this tutorial, the most popular design in the gallery is a design nam
...
 
 **Figure 05**: Replacing [ProjectName] with ContactManager ([Click to view full-size image](iteration-2-make-the-application-look-nice-cs/_static/image10.png))
 
 After you complete these steps, your web application will use the new design. The page in Figure 6 illustrates the appearance of the Contact Manager application with the October design.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-cs/_static/image6.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image11.png)
+[![Screenshot shows ContactManager with the October template.](iteration-2-make-the-application-look-nice-cs/_static/image6.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image11.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:79]: Space between Contact and Manager

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md [hunk 7]
```diff
@@ -41,15 +41,15 @@ We build the application over multiple iterations. With each iteration, we gradu
 
 The goal of this iteration is to improve the appearance of the Contact Manager application. Currently, the Contact Manager uses the default ASP.NET MVC view master page and cascading style sheet (see Figure 1). These don t look bad, but I don t want the Contact Manager to look just like every other ASP.NET MVC website. I want to replace these files with custom files.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-vb/_static/image1.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image1.png)
+[![Screenshot shows the default appearance of an ASP.NET MVC Application.](iteration-2-make-the-application-look-nice-vb/_static/image1.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image1.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:44]: "ASP dot net"

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md [hunk 8]
```diff
@@ -41,15 +41,15 @@ We build the application over multiple iterations. With each iteration, we gradu
...
 
 **Figure 01**: The default appearance of an ASP.NET MVC Application ([Click to view full-size image](iteration-2-make-the-application-look-nice-vb/_static/image2.png))
 
 In this iteration, I discuss two approaches to improving the visual design of our application. First, I show you how to take advantage of the ASP.NET MVC Design gallery to download a free ASP.NET MVC design template. The ASP.NET MVC Design gallery enables you to create a professional web application without doing any work.
 
 I decided to not use a template from the ASP.NET MVC Design gallery for the Contact Manager application. Instead, I had a custom design created by a professional design firm. In the second part of this tutorial, I explain how I worked with a professional design company to create the final ASP.NET MVC design.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-vb/_static/image2.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image3.png)
+[![Screenshot shows the ASP.NET MVC Design Gallery.](iteration-2-make-the-application-look-nice-vb/_static/image2.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image3.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:52]: "ASP dot net"

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md [hunk 9]
```diff
@@ -62,21 +62,21 @@ As I write this tutorial, the most popular design in the gallery is a design nam
...
 
 **Figure 03**: Unblocking a file downloaded from the web ([Click to view full-size image](iteration-2-make-the-application-look-nice-vb/_static/image6.png))
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-vb/_static/image4.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image7.png)
+[![Screenshot shows overwriting files in the Solution Explorer.](iteration-2-make-the-application-look-nice-vb/_static/image4.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image7.png)
 
 **Figure 04**: Overwriting files in the Solution Explorer ([Click to view full-size image](iteration-2-make-the-application-look-nice-vb/_static/image8.png))
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-vb/_static/image5.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image9.png)
+[![Screenshot shows replacing ProjectName with ContactManager.](iteration-2-make-the-application-look-nice-vb/_static/image5.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image9.png)
```
Comment (jmart1428) [inline aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:73]: Space between Project and Name and also Contact and Manager

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md (hunk 1 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md:128]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-vb.md (hunk 2 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-vb.md:128]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md (hunk 3 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:44]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md (hunk 4 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:52]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md (hunk 5 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:73]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md [hunk 10]
```diff
@@ -88,7 +88,7 @@ However, you might need to create a completely unique design for your website. I
 
 I zipped up the Contact Manager from Iteration #1 and sent the project to the design company. They did not own Visual Studio (shame on them!), but that didn't present a problem. They were able to download Microsoft Visual Web Developer for free from the [https://www.asp.net](https://www.asp.net) website and open the Contact Manager application in Visual Web Developer. In a couple of days, they had produced the design in Figure 7.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-cs/_static/image7.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image13.png)
+[![Screenshot shows the ASP.NET MVC Contact Manager Design.](iteration-2-make-the-application-look-nice-cs/_static/image7.jpg)](iteration-2-make-the-application-look-nice-cs/_static/image13.png)
```
[the remark this answers is not in the document]
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-cs.md:91]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md (hunk 7 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:44]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md (hunk 9 above)
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:73]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md [hunk 11]
```diff
@@ -62,21 +62,21 @@ As I write this tutorial, the most popular design in the gallery is a design nam
...
 
 **Figure 05**: Replacing [ProjectName] with ContactManager ([Click to view full-size image](iteration-2-make-the-application-look-nice-vb/_static/image10.png))
 
 After you complete these steps, your web application will use the new design. The page in Figure 6 illustrates the appearance of the Contact Manager application with the October design.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-vb/_static/image6.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image11.png)
+[![Screenshot shows ContactManager with the October template.](iteration-2-make-the-application-look-nice-vb/_static/image6.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image11.png)
```
[the remark this answers is not in the document]
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:79]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md [hunk 12]
```diff
@@ -88,7 +88,7 @@ However, you might need to create a completely unique design for your website. I
 
 I zipped up the Contact Manager from Iteration #1 and sent the project to the design company. They did not own Visual Studio (shame on them!), but that didn't present a problem. They were able to download Microsoft Visual Web Developer for free from the [https://www.asp.net](https://www.asp.net) website and open the Contact Manager application in Visual Web Developer. In a couple of days, they had produced the design in Figure 7.
 
-[![The New Project dialog box](iteration-2-make-the-application-look-nice-vb/_static/image7.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image13.png)
+[![Screenshot shows the ASP.NET MVC Contact Manager Design.](iteration-2-make-the-application-look-nice-vb/_static/image7.jpg)](iteration-2-make-the-application-look-nice-vb/_static/image13.png)
```
[the remark this answers is not in the document]
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-2-make-the-application-look-nice-vb.md:91]: Addressed.

File: aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md [hunk 13]
```diff
@@ -81,7 +81,7 @@ ASP.NET MVC Web Application Project. Launch Visual Studio and select the menu op
 
 Make sure that you have .NET Framework 3.5 selected from the dropdown list at the top right of the **New Project** dialog. Otherwise, the ASP.NET MVC Web Application template won't appear.
 
-[![The New Project dialog box](iteration-1-create-the-application-cs/_static/image1.jpg)](iteration-1-create-the-application-cs/_static/image1.png)
+[![Screenshot shows the New Project dialog box with ASP.NET MVC Web Application selected.](iteration-1-create-the-application-cs/_static/image1.jpg)](iteration-1-create-the-application-cs/_static/image1.png)
```
[the remark this answers is not in the document]
Comment (mSarts) [reply aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md]: Addressed.

**Pull request merged**